pkgs/top-level/all-packages.nix

```diff
@@ -10178,6 +10178,15 @@ with pkgs;
 
   inherit (nodePackages) thelounge;
 
+  theLoungePlugins = with lib; let
+    pkgs = filterAttrs (name: _: hasPrefix "thelounge-" name) nodePackages;
+    getPackagesWithPrefix = prefix: mapAttrs' (name: pkg: nameValuePair (removePrefix ("thelounge-" + prefix + "-") name) pkg) pkgs;
+  in
+  {
+    plugins = getPackagesWithPrefix "plugin";
+    themes = getPackagesWithPrefix "theme";
+  };
```

Comment:
```
$ nix-build -A theLoungePlugins.plugins.
closepms                                  thelounge-theme-common                    thelounge-theme-light                     thelounge-theme-onedark                 
giphy                                     thelounge-theme-crypto                    thelounge-theme-midnight                  thelounge-theme-purplenight             
shortcuts                                 thelounge-theme-discordapp                thelounge-theme-mininapse                 thelounge-theme-scoutlink               
thelounge-theme-abyss                     thelounge-theme-dracula                   thelounge-theme-monokai-console           thelounge-theme-seraphimrp              
thelounge-theme-amoled                    thelounge-theme-dracula-official          thelounge-theme-mortified                 thelounge-theme-solarized               
thelounge-theme-amoled-sourcecodepro      thelounge-theme-flat-blue                 thelounge-theme-neuron-fork               thelounge-theme-solarized-fork-monospace
thelounge-theme-bdefault                  thelounge-theme-flat-dark                 thelounge-theme-new-morning               thelounge-theme-zenburn                 
thelounge-theme-bmorning                  thelounge-theme-gruvbox                   thelounge-theme-new-morning-compact       thelounge-theme-zenburn-monospace       
thelounge-theme-chord                     thelounge-theme-hexified                  thelounge-theme-nologo                    thelounge-theme-zenburn-sourcecodepro   
thelounge-theme-classic                   thelounge-theme-ion                       thelounge-theme-nord                                              

$ nix-build -A theLoungePlugins.themes.
abyss                       common                      gruvbox                     mortified                   purplenight                 thelounge-plugin-shortcuts
amoled                      crypto                      hexified                    neuron-fork                 scoutlink                   zenburn                   
amoled-sourcecodepro        discordapp                  ion                         new-morning                 seraphimrp                  zenburn-monospace         
bdefault                    dracula                     light                       new-morning-compact         solarized                   zenburn-sourcecodepro     
bmorning                    dracula-official            midnight                    nologo                      solarized-fork-monospace                              
chord                       flat-blue                   mininapse                   nord                        thelounge-plugin-closepms                             
classic                     flat-dark                   monokai-console             onedark                     thelounge-plugin-giphy
```

Looks like you found more themes (and plugins?) than you bargained for?

Reply:
Fixed.
```
nix-repl> builtins.attrNames theLoungePlugins.plugins
[ "closepms" "giphy" "shortcuts" ]

nix-repl> builtins.attrNames theLoungePlugins.themes
[ "abyss" "amoled" "amoled-sourcecodepro" "bdefault" "bmorning" "chord" "classic" "common" "crypto" "discordapp" "dracula" "dracula-official" "flat-blue" "flat-dark" "gruvbox" "hexified" "ion" "light" "midnight" "mininapse" "monokai-console" "mortified" "neuron-fork" "new-morning" "new-morning-compact" "nologo" "nord" "onedark" "purplenight" "scoutlink" "seraphimrp" "solarized" "solarized-fork-monospace" "zenburn" "zenburn-monospace" "zenburn-sourcecodepro" ]
```